include/aikido/perception/ObjectDatabase.hpp

```diff
@@ -41,10 +47,17 @@ class ObjectDatabase
 
   virtual ~ObjectDatabase() = default;
 
+  /// Get the object name, resource, and offset from database by objectKey
+  /// \param[in]  objectKey: The key(string) of an object in ObjectDatabase
+  /// \param[out] objectName: The retrieved object name from ObjectDatabase
+  /// \param[out] objectResource: The retrieved uri of the object
+  /// \param[out] objectOffset: The retrieved offset matrix of the object
+  ///     e.g. the offset between a tag and the actual origin of an object
```

Comment:
I think this should be `i.e.` rather than `e.g.`.  Also, I think we should be calling this a transform rather than an offset, if we're using `Isometry3d`.